paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -333,6 +339,65 @@ class PaymentSheet internal constructor(
         }
     }
 
+    @Parcelize
+    data class PrimaryButton(
+        // Describes the colors used while the system is in light mode
+        val colorsLight: PrimaryButtonColors = PrimaryButtonColors.defaultLight,
+        // Describes the colors used while the system is in dark mode
+        val colorsDark: PrimaryButtonColors = PrimaryButtonColors.defaultDark,
+        // Describes the shape of the primary button.
+        val shape: PrimaryButtonShape = PrimaryButtonShape(),
+        // Describes the typography of the primary button.
+        val typography: PrimaryButtonTypography = PrimaryButtonTypography()
+    ) : Parcelable
+
+    @Parcelize
+    data class PrimaryButtonColors(
+        // The background color of the primary button
+        // Note: if 'null' the Appearance.Colors.primary is used.
+        @ColorInt
+        val background: Int?,
+        // The color appearing on the primary button
+        @ColorInt
+        val onBackground: Int,
+        // The border color of the primary button
```

Comment:
Here and elsewhere: do those actually end up as documentation or should they be changed to `/** This kind of comment */`

Reply:
It does not. but I changed them across this file.